dub.json

```diff
@@ -13,7 +13,7 @@
     "stringImportPaths": ["resources"],
 
     "preGenerateCommands": [
-        "git describe --tags > resources/VERSION"
+        "cd $PACKAGE_DIR && git describe --tags > resources/VERSION"
```

Comment:
@MartinNowak will this fix building out-of-tree as you commented here:  https://github.com/jacob-carlborg/dstep/pull/105#issuecomment-318119441?